python/psc/pscadios2.py

```diff
@@ -91,7 +92,13 @@ def open_store_variable(self, name, tpl):
         return xarray.DataArray(data, dims=dims, coords=coords)
 
     def get_attrs(self):
-        return {}
+        # FIXME this is not the best way to get attributes
+        def expandAttr(attr):
+            data = attr.Data()
+            if len(data) == 1:
+                return data[0]
+            return data
```

Comment:
It'd be nicer to handle scalar vs array on the adios2 side, but not sure this distinction is  actually handled properly in adios2.
Anyway, fine for now.